.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "pnpm" # See documentation for possible values
+    directory: "/web" # Location of package manifests
+    schedule:
+      interval: "monthly"
+
```

```suggestion
version: 2
updates:
  - package-ecosystem: "pnpm" # See documentation for possible values
    directory: "/web" # Location of package manifests
    schedule:
      interval: "monthly"
		labels:
			- "ok-to-test"
```
Dependabot is not part of the org. we make sure the tests are automatically triggered for any PR.

---

Oh, neat pattern! Added, I think this should be fine (but we should still manually LGTM imo).